docs/docs/manual/installing.md

```diff
@@ -190,12 +190,33 @@ brew install --cask openrefine
 
 <TabItem value="linux">
 
-Once you have downloaded the `.tar.gz` file, open a shell, navigate to the folder containing the download, and type:
+Once you have downloaded the `.tar.gz` file, open the command line (e.g., "Terminal") and navigate to the directory where the file is located.
+
+For instance, if the file is in your `Downloads` directory, use the `cd` command to navigate there.
 
 ```
-tar xzf openrefine-linux-3.4.tar.gz
+user@machine:~$ cd Downloads/
 ```
-
+Next, extract the `.tar.gz` file. To extract to the same directory, use the following command:
+  
+```
+user@machine:~$ tar xzf openrefine-3.6.2-linux.tar.gz
+```
+If you want OpenRefine to be housed in a different directory on your system, add the `-C` argument to the above command and specify the directory:
+```
+user@machine:~$ tar xzf openrefine-3.6.2-linux.tar.gz user@machine:~$ tar xzf -C /home/user/Documents
+```
+After that, use the `cd` command to navigate to the directory housing OpenRefine:
```

Comment:
@bhonick We have separate pages for `running.md` versus the `installing.md`.  Could you move your running part changes into the `running.md` file please? We have a Linux tab for it as well on that `running.md` page.  Thanks!

Reply:
We "could" entertain the idea of moving the "Starting and Exiting" portion from `running.md` into the `installing.md` if @ostephens thinks that might flow well?

Reply:
I think it is fair to lead the user up to the refine script in the install section and leave the finer configuration to the Running section, no?

Reply:
@thadguidry Does @wetneb's comment here resolve what you propose, or would you prefer that the addition to `instaling.md` to be solely about installation?

Reply:
I think he's saying that (up to the refine script, but not detailing running it, which is detailed in `running.md`).  He would need to clarify.

Reply:
@thadguidry Thanks! I've removed the section on running OpenRefine.